go-apps/meep-virt-engine/main.go

```diff
@@ -45,13 +45,23 @@ func main() {
 		run = false
 	}()
 
-	err := server.VirtEngineInit()
+	// Initialize Virt Engine
+	err := server.Init()
 	if err != nil {
-		log.Error("Failed to initialize Virt. Engine")
+		log.Error("Failed to initialize Virt. Manager")
```

Comment:
Why did we change to "virt manager" --- while rest of code is virt engine.

Reply:
Fixed.